app/src/main/java/com/project/mobility/view/activities/OnboardingActivity.java

```diff
@@ -110,6 +113,11 @@ public void goToNextPage() {
         viewPager.setCurrentItem(currentPage + 1, true);
```

Comment:
Rename goToNextPage as navigateForward() then to be consistent with navigateBackward()